app/models/SubmissionPresentation.scala

```diff
@@ -32,4 +32,12 @@ case class SubmissionPresentation(
 
 object SubmissionPresentation {
   implicit val formats = Json.format[SubmissionPresentation]
+
+  object ActionTypes {
+    val Arrival = "Arrival"
```

Comment:
Why strings? Maybe case objects with common `sealed abstract class`

Reply:
Yea, good idea. Changing that.